src/main/scala/devices/debug/dm_registers.scala

```diff
@@ -1,6 +1,6 @@
 package freechips.rocketchip.devices.debug
 
-import Chisel._
+import chisel3._
 
 // This file was auto-generated from the repository at https://github.com/riscv/riscv-debug-spec.git,
```

Comment:
we should fix the source script (if it is still used!) I'll make a PR for that.

Reply:
https://github.com/riscv/riscv-debug-spec/pull/502

Reply:
thanks, I hadn't gotten to that yet :-)